doc/man1/openssl-s_time.pod.in

```diff
@@ -162,6 +173,10 @@ be able to measure the performance of all protocols with all servers.
 The B<-verify> option should really exit if the server verification
 fails.
 
+=head1 HISTORY
+
+The B<-cafile> option was deprecated in OpenSSL 3.0.
```

Comment:
Side note: Should there be a way of formally marking options that are deprecated, e.g. so that they appear as such in "-help" output, and disappear in no-deprecated builds? Not for this PR.

Reply:
Created issue #10956